packages/chat-stateful-client/src/types/ChatMessageWithStatus.ts

```diff
@@ -17,4 +17,6 @@ export type ChatMessageWithStatus = ChatMessage & {
   status: MessageStatus;
   /* @conditional-compile-remove(data-loss-prevention) */
   policyViolation?: boolean;
+  /* @conditional-compile-remove(teams-inline-images-and-file-sharing) */
+  resourceCache?: Record<string, string>;
```

Comment:
Should we add some documentation for this prop?

Reply:
We can add the documentation in the follow up to the storybook updates